homeassistant/components/calendar/__init__.py

```diff
@@ -137,7 +137,7 @@ def event(self):
         raise NotImplementedError()
 
     @property
-    def state_attributes(self):
+    def extra_state_attributes(self):
```

Comment:
This is an entity integration with platforms so should have final state_attributes.

Reply:
Fixed